src/components/Incorporation/IncorporationDateTime.vue

```diff
@@ -241,33 +243,33 @@ export default class IncorporationDateTime extends Mixins(DateMixin) {
   }
 
   /** The minimum date that can be entered (today). */
-  private get minDate (): string {
+  get minDate (): string {
     return this.dateToYyyyMmDd(this.getCurrentJsDate)
   }
 
   /** The maximum date that can be entered (today + 10 days). */
-  private get maxDate (): string {
+  get maxDate (): string {
     const date = new Date(this.getCurrentJsDate) // make a copy
     date.setDate(date.getDate() + 10)
     return this.dateToYyyyMmDd(date)
   }
 
   /** The minimum time that can be entered (now + 3 minutes). */
-  private minTime (): string {
+  protected minTime (): string {
     const date = this.getCurrentJsDate
     return new Date(date.getTime() + 180000)
       .toLocaleTimeString('en-US', { hour: '2-digit', minute: '2-digit', hour12: true })
   }
```

Comment:
This could probably be a getter instead of a function but that's _really_ out of scope for this cleanup.